rust/lance-datafusion/src/exec.rs

```diff
@@ -45,7 +49,12 @@ impl OneShotExec {
         let schema = stream.schema().clone();
         Self {
             stream: Mutex::new(Some(stream)),
-            schema,
+            schema: schema.clone(),
+            properties: PlanProperties::new(
+                EquivalenceProperties::new(schema),
+                Partitioning::RoundRobinBatch(1),
+                datafusion::physical_plan::ExecutionMode::Bounded,
+            ),
         }
```

Comment:
The `ExecPlan` interface changed.  `output_partitioning` and `output_ordering` have gone away and `properties()` now replaces both of those (as well as a new field, "execution mode".